prototype_source/max_autotune_CPU_with_gemm_template_tutorial.rst

```diff
@@ -0,0 +1,198 @@
+Max-autotune Support on CPU with GEMM Template Tutorial
+==============================================================
+
+**Author**: `Jiong Gong <https://github.com/jgong5>`__, `Leslie Fang <https://github.com/leslie-fang-intel>`__, `Chunyuan Wu <https://github.com/chunyuan-w>`__
+
+Prerequisites:
+----------------
+-  `torch.compile and TorchInductor concepts in PyTorch <https://pytorch.org/tutorials/intermediate/torch_compile_tutorial.html>`__
+
+Introduction
```

Comment:
Perhaps, add the RFC link in the intro as the reference too?

Reply:
RFC link has been added